mediator/med_phases_prep_atm_mod.F90

```diff
@@ -115,8 +115,7 @@ subroutine med_phases_prep_atm(gcomp, rc)
     !--- map atm/ocn fluxes from ocn to atm grid if appropriate
     !---------------------------------------
     if (trim(coupling_mode) == 'cesm' .or. &
-        trim(coupling_mode) == 'nems_frac_aoflux' .or. &
-        trim(coupling_mode) == 'nems_frac_aoflux_sbs') then
```

Comment:
@DeniseWorthen The agrid test is the fully coupled not side-by-side. You could see the coupling mode in here,

https://github.com/ufs-community/ufs-weather-model/blob/fb788ba57a7c7d8f318003a9219c4d5044ad85fe/tests/tests/cpld_control_noaero_p8_agrid#L87C16-L87C32

It is hard to remember but if you want to perform side-by-side run, you need to change the coupling mode as `nems_frac_aoflux_sbs` and that will prevent feedback from CMEPS to FV3.

Reply:
I see.  The ``nems_frac_aoflux_sbs`` always is checked along with ``nems_frac_aoflux`` in logical statements, so I think it should still work.

Reply:
@DeniseWorthen Probably. Slide 10 in the following presentation has some information about side-by-side configuration. https://docs.google.com/presentation/d/15MZA6VMzJ9vnR3RV5nj4feuRxtqjOC4LdCsJqzDbVsM/edit#slide=id.g13117be911c_0_80

Reply:
@DeniseWorthen I think it would be nice to test side-by-side with current develop. I am looking to the old PR https://github.com/NOAA-EMC/CMEPS/compare/763405c02a8f7eec8c1e1a360fcb5bf3570604e1...424a08638dc34828b2aa64fbb75e7f368f52bf3d and I am seeing some difference in the code with the current head of develop. Especially in the `[esmFldsExchange_nems_mod.F90](https://github.com/NOAA-EMC/CMEPS/pull/101#diff-ebf9a9f9de2a06b3ec74f8dc021a9f53296758b1d6d76498a0fc76f77fe607bf)` and also `med_merge_auto ` call in `mediator/med_phases_prep_atm_mod.F90`. So, maybe side-by-side case is broke at this point.

Reply:
The agrid test in this PR with ``use_med_flux=.false.`` reproduces the develop branch with ``use_med_flux=.false.`` 

The agrid test w/ sbs turned on does not reproduce the non-sbs agrid in either the develop branch or this PR branch, which is expected. Is that right?

Reply:
@DeniseWorthen Yes. That is expected. I am expecting to have that `cpld_control_noaero_p8_agrid` and `cpld_control_noaero_p8` must have identical results (if there is not any configuration changes defined in the test file - sometimes it happens since cpld_control_noaero_p8  involving but cpld_control_noaero_p8_agrid is not sync with it) but in terms of coupling (i.e. sfcf and atmf files needs to be identical since side-by-side does not send the fluxes to FV3).

Reply:
Ok, let me confirm that was still true in develop, which should mean it is still true in this PR branch.

Without a test for agrid_sbs, there is no guarantee that it won't break at some point.

Reply:
@DeniseWorthen Yes. You are right. It is hard to be sure that side-by-side is still workin. Once ESMF 8.5.0 is available I'll test xgrid reproducibility issue and if we change agrid to xgrid in that time, I could also look at possible issues with sbs.

Reply:
I don't think I'm testing the sbs correctly. The agrid test uses the sfcocn suite, but all other coupled tests use the suite containing NST. 

In any case, I know that the current PR reproduces all the tests in the develop branch at this point.

Reply:
@DeniseWorthen Okay. That is fine. I think NNST is not an option for CMEPS since I think it requires additional input (I am not sure about other dependencies in the upstream ccpp steps). Anyway, I could look at in a more detailed way once 8.5.0 available. Thanks for checking.